aiosmtpd/tests/test_handlers.py

```diff
@@ -1,9 +1,11 @@
+import asyncio
```

Comment:
As long as `tox -e qa` passes, or at least doesn't throw any new warnings over master, I'm happy.

(Note that I'm hoping to improve this as soon as the next version of flake8 comes out.)